packages/php-wasm/universal/src/lib/php-request-handler.ts

```diff
@@ -361,7 +361,11 @@ export function seemsLikeAPHPRequestHandlerPath(path: string): boolean {
 }
 
 function seemsLikeAPHPFile(path: string) {
-	return path.endsWith('.php') || path.includes('.php/');
+	return (
+		path.endsWith('.php') ||
+		path.includes('.php/') ||
+		path.match(/sitemap.*.xml$/) != null
```

Comment:
Why is this change needed? This could cause issues with requesting XML files as they could end up being run as PHP files instead of treated as static HTML.